frontend/components/EnrollSecrets/DeleteSecretModal/DeleteSecretModal.tsx

```diff
@@ -51,7 +51,7 @@ const DeleteSecretModal = ({
           </p>
           <p>
             Any enrolled hosts using this secret will not receive updates
```

Comment:
@lucasmrod is this really how it works today? Meaning, if I'm trying to rotate a compromised secret, I have to redeploy fleetd. Otherwise, I won't be able to update agent options for my hosts

If this is true, I think there's an opportunity to reduce the language here:

"Hosts that enrolled with this secret will not get updates to agent options."

Reply:
@noahtalerman while working on this PR I made a feature request to revisit the copy in this modal (there is a lot of text and some of it feels repetitive)

Will definitely make this change in the mean time 👍